docs/assets/swagger-template.hbs

```diff
@@ -1,5 +1,4 @@
 ---
-title: Endpoints
 parent: API
 grand_parent: estela
 ---
```

Comment:
All the lines between the dashes `---` should be removed.

Reply:
Done